templates/inspec/examples/attributes/google_compute_ssl_policies.erb

```diff
@@ -1,2 +1,2 @@
-project_name = attribute(:gcp_project_name, default: '<%= grab_attributes['project_name'] -%>')
+project_name = attribute(:gcp_project_name, default: 'chef-gcp-inspec')
```

Comment:
Why is this now a constant?

Reply:
This property is declared in the existing inspec integration tests. I am using whatever is set in the integration test config file with this attribute, but that means that magic modules doesn't have its own declaration of it.